classes/system/ceilometer/server/cluster.yml

```diff
@@ -7,12 +7,12 @@ parameters:
     server:
       database:
         influxdb:
-          host: ${_param:monitoring_service_address}
+          host: ${_param:stacklight_monitor_address}
```

Comment:
The influxdb host should be  `${_param:stacklight_monitor_node01_address}`. See https://github.com/Mirantis/mk-lab-salt-model/blob/4515574e43526e940bfff32a145a4ca477bbf608/classes/cluster/mk20_stacklight_advanced/stacklight/init.yml#L4.